pkg/processor/k8sprocessor/options.go

```diff
@@ -365,3 +365,34 @@ func WithDelimiter(delimiter string) Option {
 		return nil
 	}
 }
+
+var defaultExcludes = ExcludeConfig{
+	Pods: []ExcludePodConfig{
```

Comment:
As discussed, we could keep the list empty and populate it in Helm chart only for tracing pipeline